overrides/scripts/ModSpecific/RFTools.zs

```diff
@@ -217,6 +217,10 @@ recipes.addShaped(<rftools:matter_beamer>, [[<enderio:item_material:73>,<ore:glo
 recipes.remove(<rftools:syringe>);
 recipes.addShapedMirrored(<rftools:syringe>, [[<enderio:item_alloy_ingot:6>,<enderio:item_alloy_ingot:6>,null],[<enderio:item_alloy_ingot:6>,<enderio:item_soul_vial>,<contenttweaker:crystalline_brown_slime_ingot>],[null,<contenttweaker:crystalline_brown_slime_ingot>,<enderio:item_alloy_ingot:6>]]);
 
+// Filter Module
+recipes.remove(<rftools:filter_module>);
```

Comment:
i may be incorrect, but i believe there is a recipe to convert a configured filter module into a fresh one. if there isnt, there ought to be, so please add a hidden recipe to clear the filter module data